atomic_reactor/utils/cachito.py

```diff
@@ -147,14 +147,16 @@ def wait_for_request(
 
             # All other states are expected to be transient and are not checked.
 
-            elapsed = time.time() - start_time
+            updated_time = datetime.fromisoformat(response_json["updated"])
+            elapsed = (datetime.utcnow() - updated_time).total_seconds()
```

Comment:
Should we trust the time on an external service here? We could make atomic-reactor start counting from the last time it has perceived  a new update instead.

Reply:
That is interesting to consider, however, since Cachito uses Kerberos for its workers to authenticate with the REST API, there would be a Cachito outage if there was more than a 5 minute time gap.

I'll add an extra commit which implements your suggested approach. Then you and other reviewers can let me know which approach you prefer.

Reply:
@athos-ribeiro I ended up just amending the original commit since this seemed pretty clean.